src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -437,6 +437,51 @@ namespace UtilityRoutines {
     bool case_insensitive_comparator::operator()(const std::string& a, const std::string& b) const noexcept {
         return SameString(a, b);
     }
+
+    void appendPerfLog(std::string const &colHeader, std::string const &colValue, bool finalColumn) 
+    // Add column to the performance log file (comma separated) which is appended to existing log.
+    // The finalColumn (an optional argument) being true triggers the actual file to be written or appended.
+    // J.Glazer February 2020
+    {
+        static std::string headerRow = "";
+        static std::string valuesRow = "";
+
+        // the following was added for unit testing to clear the static strings
+        if (colHeader == "RESET" && colValue == "RESET") {
+            headerRow = "";
+            valuesRow = "";
+            return;
+        }
+
+        //accumuate the row until ready to be written to the file.
+        headerRow = headerRow + colHeader + ",";
+        valuesRow = valuesRow + colValue + ",";
+
+        if (finalColumn) {
+            bool FileExists;
+            {
+                IOFlags flags;
+                ObjexxFCL::gio::inquire(DataStringGlobals::outputPerfLogFileName, flags);
```

Comment:
@Myoldmopar Is there a non-gio equivalent that should be used here?

Reply:
For time's sake I suggest we leave this if it is working.  We can put a follow-up task to move this to a more standard approach after freeze.